gateway/v10.ts

```diff
@@ -428,6 +437,121 @@ export interface GatewayReadyDispatchData {
  */
 export type GatewayResumedDispatch = DataPayload<GatewayDispatchEvents.Resumed, never>;
 
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-create
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-update
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-delete
+ */
+export type GatewayAutoModerationRuleModifyDispatch = DataPayload<
+	| GatewayDispatchEvents.AutoModerationRuleCreate
+	| GatewayDispatchEvents.AutoModerationRuleUpdate
+	| GatewayDispatchEvents.AutoModerationRuleDelete,
+	GatewayAutoModerationRuleModifyDispatchData
+>;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-create
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-update
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-delete
+ */
+export type GatewayAutoModerationRuleModifyDispatchData = APIAutoModerationRule;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-create
+ */
+export type GatewayAutoModerationRuleCreateDispatch = GatewayAutoModerationRuleModifyDispatch;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-create
+ */
+export type GatewayAutoModerationRuleCreateDispatchData = GatewayAutoModerationRuleModifyDispatchData;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-update
+ */
+export type GatewayAutoModerationRuleUpdateDispatch = GatewayAutoModerationRuleModifyDispatch;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-update
+ */
+export type GatewayAutoModerationRuleUpdateDispatchData = GatewayAutoModerationRuleModifyDispatchData;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-delete
+ */
+export type GatewayAutoModerationRuleDeleteDispatch = GatewayAutoModerationRuleModifyDispatch;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-rule-delete
+ */
+export type GatewayAutoModerationRuleDeleteDispatchData = GatewayAutoModerationRuleModifyDispatchData;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-action-execution
+ */
+export type GatewayAutoModerationActionExecutionDispatch = DataPayload<
+	GatewayDispatchEvents.AutoModerationActionExecution,
+	GatewayAutoModerationActionExecutionDispatchData
+>;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway#auto-moderation-action-execution-auto-moderation-action-execution-event-fields
```

Comment:
https://discord.com/developers/docs/topics/gateway-events#auto-moderation-action-execution is the correct link.